apis/src/components/organisms/display_profile.rs

```diff
@@ -0,0 +1,97 @@
+use crate::{
+    components::molecules::game_row::GameRow,
+    functions::users::{get::get_user_games, user_response::UserResponse},
+};
+use hive_lib::game_status::GameStatus;
+use leptos::{logging::log, *};
+
+#[derive(Clone, PartialEq)]
+pub enum TabView {
+    Playing,
+    Games,
+}
+
+#[component]
+pub fn DisplayProfile(user: StoredValue<UserResponse>) -> impl IntoView {
```

Comment:
I would swap the location of playing and finished so that playing is left and finished games are on the right.